release/mkrelease.js

```diff
@@ -0,0 +1,306 @@
+const fs = require('fs');
+const request = require('request');
+const cp = require('child_process');
+const mkpath = require('mkpath');
+const naturalSort = require('natural-sort');
+
+var gitHubRequest = request.defaults({
+    headers: {'User-Agent': 'Request'}
+})
+const integrationDir = __dirname + '/../_layout/integrations';
```

Comment:
Style nit: use uppercase for consts - https://github.com/felixge/node-style-guide#use-uppercase-for-constants